Makefile

```diff
@@ -55,21 +45,17 @@ test-unit:
 test-cover:
 	${DEV_ENV_CMD} test-cover.sh
 
-update-changelog:
-	${DEV_ENV_PREFIX} -e RELEASE=${WORKFLOW_RELEASE} ${DEV_ENV_IMAGE} gen-changelog.sh \
-	  | cat - CHANGELOG.md > tmp && mv tmp CHANGELOG.md
-
 docker-build: build
 	docker build --rm -t ${IMAGE} rootfs
 	docker tag ${IMAGE} ${MUTABLE_IMAGE}
 
-# Push to a registry that Kubernetes can access.
-docker-push:
-	docker push ${IMAGE}
+check-kubectl:
+	@if [ -z $$(which kubectl) ]; then \
+		echo "kubectl binary could not be located"; \
+		exit 2; \
+	fi
 
-deploy: docker-build docker-push
-	sed 's#\(image:\) .*#\1 $(IMAGE)#' /tmp/deis-$(SHORT_NAME) | kubectl apply --validate=true -f -
-	kubectl scale rc deis-$(SHORT_NAME) --replicas 0 --namespace deis
-	kubectl scale rc deis-$(SHORT_NAME) --replicas $(DESIRED_REPLICAS) --namespace deis
+deploy: check-kubectl docker-build docker-push
```

Comment:
Nope, the `docker-push` target comes from `versioning.mk`. I'm not sure why this duplicated target is here.